cpp/src/arrow/python/numpy-internal.h

```diff
@@ -57,6 +57,7 @@ class Ndarray1DIndexer {
   T* begin() const { return data(); }
   T* end() const { return begin() + size() * stride_; }
 
+  // XXX should be `stride_ != 1`
   bool is_strided() const { return stride_ == 1; }
```

Comment:
Is this is a problem that should be fixed here?

Reply:
I don't know. Perhaps the naming is simply wrong. I should take a look where it's used.